core/modules/comment/comment.module

```diff
@@ -2137,7 +2137,6 @@ function comment_form_submit($form, &$form_state) {
     if (user_is_anonymous()) {
       user_cookie_save(array_intersect_key($form_state['values'], array_flip(array('name', 'mail', 'homepage'))));
     }
-
     comment_save($comment);
```

Comment:
Since method isNew() after saving will return false, let's use a variable to store the return value here.

 ```suggestion
    $is_new = $comment->isNew();
    comment_save($comment);
```